merge_claude_settings.sh

```
@@ -23,7 +23,32 @@ def merge:
     )
   );
 merge
-' .claude/settings.json ~/.claude/settings.local.json > ~/.claude/settings.json
+if [ -f ~/.claude/settings.local.json ]; then
+  tmp_file="$(mktemp ~/.claude/settings.json.XXXXXX)" || exit 1
+  if jq -s '
+def merge:
+  reduce .[] as $item ({}; 
```

Shellcheck (0.11.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)